scRNA-seq-advanced/04-differential_expression.Rmd

```diff
@@ -171,32 +275,27 @@ library_ids <- c(
   "SCPCL000491"
 )
 
-# subset sce to only contain samples with library IDs of interest 
+# subset sce to only contain samples with IDs of interest 
 samples_to_keep <- integrated_sce$sample %in% library_ids
 rms_sce <- integrated_sce[,samples_to_keep]
 
-# remove original integrated object 
+# remove original integrated object to save some memory
```

Comment:
For a modicum of "safety" I would move the removal to _after_ printing out the new SCE. Always check what you have before removing something that might take some time to reload/reproduce!

"Now that we know we have what we want, we can remove the old version to save memory."